### PR TITLE
Fix/pr 142/proctoring start date filter

### PR DESCRIPTION
**Pull request by zagovorichev**
Pull request summary
 
Related to: https://oat-sa.atlassian.net/browse/PR-142
 
- datetime-range replaced with date-rage as proctoring never worked with H:s (it was added by mistake when new datetime picker was added) [here](https://github.com/oat-sa/extension-tao-proctoring/pull/866/files#diff-92ee372f909de10dcd3ebee0c4539e6f9bcf7acb9ef00aca5bedbc930ba7002aR961)
- Deleted time adjustment error log, for the test sessions that can’t use it (thousands records in the log, that we don’t need) [here](https://github.com/oat-sa/extension-tao-proctoring/pull/866/files#diff-ef33637e6c90a7eda0155ee4cca8ac1f1e0b3f660cc7812e4db2dedb2c68c2bbR111)
- Issue with localization of the datetime picker, it should be initialized with datepicker component, not manually [here](https://github.com/oat-sa/extension-tao-proctoring/pull/866/files#diff-92ee372f909de10dcd3ebee0c4539e6f9bcf7acb9ef00aca5bedbc930ba7002aR965)
- line highlighting - delivery executions in the awaiting state should be highlighted, but it doesn’t work for the other then EN langs [here](https://github.com/oat-sa/extension-tao-proctoring/pull/866/files#diff-92ee372f909de10dcd3ebee0c4539e6f9bcf7acb9ef00aca5bedbc930ba7002aR1008)
 
#### Steps to reproduce  (for bugs)
__datetime-range__
 - datepicker has fields for the hours and seconds

 __logs__
- When the test taker is waiting for the approval from the proctor and proctor is on the proctoring page - there are a lot of errors in the log about the adjustment time, which generated on each datatable update

__highlights__
- delivery executions in the state AWAITING are not highlighted for the languages different from En

__datepicker__
- when the language different than En there is only 1 date in the date range picker by default
  
#### How to test
 
__datetime-range__
- there are no Hours and seconds in the date range picker

__logs__
- there are no records in the log about the adjustment time, when delivery execution can't be adjusted

__highlights__
- each language highlights delivery executions in the state PAUSE

__datepicker__
- default value of the datepicker is localized

### CODE REVIEW

File: views/js/controller/Delivery/monitoring.js [hunk 1]
```diff
@@ -965,11 +955,22 @@ define([
                                     startDatePicker.destroy();
                                 }
 
+                                console.log($elt.val())
```
Comment (andrey-ponamarev) [inline views/js/controller/Delivery/monitoring.js:958]:
```suggestion
```

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by tikhanovichA — APPROVED** [review]
Looks good! Just one small note form @andrey-ponamarev 
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/js/controller/Delivery/monitoring.js (hunk 1 above)
Comment (zagovorichev) [reply views/js/controller/Delivery/monitoring.js]: my bad, good catch, thank you!

**Pull request merged**